src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/utils/UserReceiptMailBuilder.kt

```diff
@@ -126,7 +126,7 @@ class UserReceiptMailBuilder(@Autowired private val confidentialMailUtils: Confi
                   RefNumberTemplate(Type.CODICE_AVVISO, paymentNotice.rptId().noticeId),
                   null,
                   PayeeTemplate(
-                    transactionUserReceiptData.receivingOfficeName,
+                    transactionUserReceiptData.receivingOfficeName ?: "",
```

Comment:
Add a test where receivingOfficeName is value and not checking for it's valued with blank in case it's not